shaders/compute_nbody/particle_calculate.comp

```diff
@@ -28,18 +28,18 @@ layout(std140, binding = 0) buffer Pos
    Particle particles[ ];
 };
 
-layout (local_size_x = 256) in;
-
 layout (binding = 1) uniform UBO 
 {
 	float deltaT;
 	int particleCount;
 } ubo;
 
-layout (constant_id = 0) const int SHARED_DATA_SIZE = 512;
-layout (constant_id = 1) const float GRAVITY = 0.002;
-layout (constant_id = 2) const float POWER = 0.75;
-layout (constant_id = 3) const float SOFTEN = 0.0075;
+layout (constant_id = 1) const int SHARED_DATA_SIZE = 512;
```

Comment:
Shared data size here (512) seems to be inconsistent with the defaults used above. It probably doesn't matter in practice since you always set the specialization constant, but it might be clearer if they matched?

Reply:
I chnaged the shader defaults to match the ones in the C++ files.